packages/global-styles/src/gutenberg-bridge/index.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * Unlock the private apis for the global styles related functionalities and re-export them
+ * on our own as this kind of internal apis might be drastically changed from time to time.
+ * See https://github.com/Automattic/wp-calypso/issues/77048
+ */
+import { privateApis as blockEditorPrivateApis } from '@wordpress/block-editor';
+import { createHigherOrderComponent } from '@wordpress/compose';
+import { __dangerousOptInToUnstableAPIsOnlyForCoreModules } from '@wordpress/private-apis';
+import mergeWith from 'lodash/mergeWith';
+import type { GlobalStylesObject } from '../types';
+
+const { unlock } = __dangerousOptInToUnstableAPIsOnlyForCoreModules(
+	'I know using unstable features means my plugin or theme will inevitably break on the next WordPress release.',
```

Comment:
I hadn't realized they added this message, but I'm definitely a fan since it forces us to be explicit about it 😅

Reply:
Yup, I love it too 😉 Great tactics from a devex perspective. 👍